plugins/module/create_kafka_acl_binding.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python
+
+
+from __future__ import (absolute_import, division, print_function)
+import random
+import time
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: create_kafka_acl_binding
+
+short_description: Create Access Control Lists (ACLs) for Red Hat OpenShift Streams for Apache Kafka Instance
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Create Access Control Lists (ACLs) Red Hat OpenShift Streams for Apache Kafka Instance. More details can be found [here](https://github.com/redhat-developer/app-services-sdk-python/blob/main/sdks/kafka_instance_sdk/docs/AclBinding.md)
+
+options:
+    principal:
+        description: ID of the Service Account to bind created ACLs to
```

Comment:
This description is invalid. Please take look into CLI how we document principal

Reply:
Will look into it. Cheers for this. 🤙